pipeline_steps/kibana.groovy

```diff
@@ -62,7 +62,9 @@ def kibana_tests(vm=null){
         export PATH=\$PATH:./phantomjs-2.1.1-linux-x86_64/bin
         # Remove any existing screenshots from old runs
         rm -f *.png
-        python conf-gen.py --secure --password-file /etc/openstack_deploy/user_rpco_secrets.yml \
+
+        export PASSWORD=\$(grep -Ir kibana_password /etc/openstack_deploy/ | head -1 | sed 's/.*\\: //')
```

Comment:
This seems to work as expected. It looks like you could have changed this to the following to achieve the same thing:
`python conf-gen.py --secure --password-file /etc/openstack_deploy`

I would have thought `kibana_password` would only be set once, your use of `head -1` implies otherwise. If it was defined multiple times I think the last would have been used by Ansible and not the first. That said, it would probably be better if it failed when the password was set multiple times given that shouldn't happen.